src/helpers/daily-data-types.tsx

```diff
@@ -16,6 +16,7 @@ export enum DailyDataType {
 	AppleHealthInBedMinutes = "AppleHealthInBedMinutes",
 	AppleHealthStandMinutes = "AppleHealthStandMinutes",
 	AppleHealthSteps = "AppleHealthSteps",
+	AppleHealthStepsWhileWearingWatch = "AppleHealthStepsWhileWearingWatch",
```

Comment:
Maybe `AppleHealthStepsWhileWearingDevice` just because it could be some other kind of wearable that triggers it to count.

Reply:
Good idea.  Done.